src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_TravelAidScreen.tsx

```diff
@@ -0,0 +1,120 @@
+  const styles = useStyles();
+  const {t} = useTranslation();
+  const {enable_travel_aid_stop_button} = useRemoteConfig();
+  const {setPreference, preferences} = usePreferences();
+  const {contactPhoneNumber} = useFirestoreConfiguration();
+
+  const hasContactPhoneNumber = !!contactPhoneNumber;
+
+  const travelAidToggleTitle = t(TravelAidSettingsTexts.toggle.title);
+  const travelAidSubtext = t(TravelAidSettingsTexts.toggle.subText);
+
+  const toggleValue = !enable_travel_aid_stop_button
+    ? false
+    : preferences.journeyAidEnabled;
+
+  return (
+    <FullScreenView
+      headerProps={{
+        title: t(TravelAidSettingsTexts.header.accessibility.title),
+        leftButton: {type: 'back', withIcon: true},
+      }}
+      parallaxContent={(focusRef) => (
+        <ScreenHeading
+          ref={focusRef}
+          text={t(TravelAidSettingsTexts.header.accessibility.title)}
+        />
+      )}
+    >
+      <View style={styles.content}>
+        <Section>
+          <ToggleSectionItem
+            text={travelAidToggleTitle}
+            value={toggleValue}
+            onValueChange={(checked) =>
+              setPreference({journeyAidEnabled: checked})
+            }
+            subtext={travelAidSubtext}
+            disabled={!enable_travel_aid_stop_button}
+            isSubtextMarkdown
+            testID="toggleTravelAid"
+          />
+          <GenericSectionItem style={styles.buttonContainer}>
+            <View style={styles.buttonContainer}>
+              <Button
+                mode="secondary"
+                backgroundColor="background_0"
+                text={t(TravelAidSettingsTexts.button.importantInfo.title)}
+                accessibilityHint={t(
+                  TravelAidSettingsTexts.button.importantInfo.a11yHint,
+                )}
+                accessibilityRole="button"
+                testID="travelAidImportantInformationButton"
+                onPress={() => {
+                  navigation.navigate('Profile_TravelAidInformationScreen');
+                }}
+              />
+              {hasContactPhoneNumber && (
+                <Button
+                  mode="secondary"
+                  backgroundColor="background_0"
+                  text={t(TravelAidSettingsTexts.button.contact.title)}
+                  accessibilityHint={t(
+                    TravelAidSettingsTexts.button.contact.a11yHint,
+                  )}
+                  accessibilityRole="button"
+                  testID="travelAidContactCustomerServiceButton"
+                  onPress={async () => {
+                    const phoneNumber = `tel:${contactPhoneNumber}`;
+                    if (await Linking.canOpenURL(phoneNumber)) {
+                      Linking.openURL(phoneNumber);
+                    } else {
+                      Bugsnag.notify(
+                        new Error(
+                          'Could not open phone number in accessiblity settings',
```

Comment:
Could add the phone number to the error msg here, so we see it if there is something wrong with our logic and the link can't be opened because the phone number is invalid.

Reply:
Added.